src/adstage_api_concepts/hal_json/embedded_resources.apib

```diff
@@ -0,0 +1,52 @@
+## Embedded Resources
+
+When issuing a request to a HAL endpoint, the requested resource will typically appear inside the top-level JSON response body.
+However, the response body may also contain `_embedded` objects containing information about resources related to the parent resource.
+For example, when requesting information about a user in example below, the main JSON body returns data about that user, and the `_embedded` key returns information about the user's organization.
+
+Note that any particular resource returned in the `_embedded` object may have additional HAL resources, links, and forms of its own.
+
+**Modified HAL response from a request to a user data endpoint**
+
+```
+{
+    // The main response contains data about the user resource,
+    // which was the item requested from the endpoint
+    "admin": false,
```

Comment:
Let's avoid calling attention to this flag. We should honestly probably omit this flag unless the user is actually an admin.

Reply:
Agree - I thought I made a ticket for this somewhere, but maybe I just brought it up in a 1-on-1 or something. But I think it would be interesting to see just how far someone could hack into undocumented APIs just by following HAL links and making reasonable assumptions...

Reply:
Though I did also add language in the docs discouraging that

Reply:
- [x] remove